src/main/java/uk/gov/dwp/dataworks/util/ArrayUtils.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.dwp.dataworks.util;
+
+public class ArrayUtils {
+    public static void reverse(byte[] array) {
```

Comment:
I'd probably make this return a string rather than editing the input directly. It's effectively an input/output arg then which isn't really clean code.

Reply:
Discussed with @nicholasjgreen that this definitely isn't lovely, but we'll leave for now